library/weno/src/TransmitData.php

```diff
@@ -86,4 +89,35 @@ public function validatePatient($pid)
          $patientRes = sqlQuery($patientInfo, $val);
          return $patientRes;
     }
+
+    public function validateNPI($npi)
+    {
+
+        $headers = array(
+            'Content-Type' => "application/json"
+        );
+        $query = [
+            'number' => $npi,
+            'enumeration_type' => '',
+            'taxonomy_description' => '',
+            'first_name' => '',
+            'last_name' => '',
+            'organization_name'  => '',
+            'address_purpose' => '',
+            'city' => '',
+            'state' => '',
+            'postal_code' => '',
+            'country_code' => '',
+            'limit' => '',
+            'skip' => '',
+            'version' => '2.0',
+        ];
+        $response = oeHttp::bodyFormat('body')->get('https://npiregistry.cms.hhs.gov/api/', $query);
+
+        $body = $response->body(); // already should be json.
+
+        $validated = json_decode($body);
+
+        return $validated->result_count;
```

Comment:
What is "->result_count" supposed to be doing? I tried to google search this and couldn't figure it out.

Reply:
Go here https://npiregistry.cms.hhs.gov/api/demo?version=2.0

When you look at the URL the first thing in the URL is the result count. If the result count is 1 that means that the NPI number was found with all the other data that goes with it. We could actually use this API to keep the pharmacy information up to date if we wanted.

Reply:
hi @juggernautsei , Very nifty; learn something new every day; found an example of this at bottom of : https://stackoverflow.com/questions/12429029/php-get-values-from-json-encode